spec/features/state_file/complete_intake_spec.rb

```diff
@@ -82,6 +84,7 @@
       expect(page).to have_text "You have successfully submitted your taxes"
       click_on "Show XML"
       expect(page.body).to include('efile:ReturnState')
+      expect(page.body).to include('<FirstName>Testy</FirstName>')
```

Comment:
again maybe not needed but validating we're still getting user data back with encryption